src/components/__tests__/change-view.test.jsx

```diff
@@ -0,0 +1,356 @@
+/* eslint-env jest */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import ChangeView, {defaultDiffType} from '../change-view/change-view';
+import layeredStorage from '../../scripts/layered-storage';
+import simplePage from '../../__mocks__/simple-page.json';
+import WebMonitoringDb from '../../services/web-monitoring-db';
+import {diffTypesFor} from '../../constants/diff-types';
+
+jest.mock('../../scripts/layered-storage');
+
+const mockChange = {
+  uuid: 'adc70e01-0723-4a28-a8e4-ca4b551ed2ae..0772dc83-7966-4881-8901-eceb051b9536',
+  uuid_from: 'adc70e01-0723-4a28-a8e4-ca4b551ed2ae',
+  uuid_to: '0772dc83-7966-4881-8901-eceb051b9536',
+  priority: null,
+  current_annotation: {},
+  created_at: null,
+  updated_at: null,
+  significance: null
+};
+
+// Note for testing relevant diffTypes:
+// the defaultDiffType (SIDE_BY_SIDE_RENDERED) is only relevant for the "text/html" media type
+// the relevant types for "text/*" are a subset of the relevant types for "text/html"
+// the relevant types for 'text/html' and "*/*" are mutually exclusive
+
+const diffTypeStorage = 'edgi.wm.ui.diff_type';
```

Comment:
This should probably be being exported from `ChangeView`, too, right?

In the long run, maybe we should have a Preferences or Settings class that the app provides via context, and that provides clear slots for all these things. Dunno. Anyway, that’s out of scope here.